src/main/webapp/portal-core/js/portal/layer/querier/QueryTargetHandler.js

```diff
@@ -120,17 +120,53 @@ Ext.define('portal.layer.querier.QueryTargetHandler', {
                 continue;
             }
 
-
             point = Ext.create('portal.map.Point', {
                 latitude : queryTargets[i].get('lat'),
                 longitude : queryTargets[i].get('lng')
             });
 
+
             var shortTitle = cswRecord.get('name');
+            
+            // abbreviate the service provider name
+            var provider = cswRecord.get('contactOrg');   
+            
+            // list of service providers 
+            var SERVICE_PROVIDERS = [
+                'CSIRO',
+                'Geoscience Australia',
+                'Northern Territory',
+                'Queensland',
+                'South Australia',
+                'Tasmania',
+                'Victoria',
+                'Western Australia'
+            ];
+            
+            for (var i = 0; i < SERVICE_PROVIDERS.length; i++) {                
```

Comment:
But it is a contains relationship that I am looking for so I still need to loop through the map entries, right? ie the value is not the key but might contain the key phrase. I can't see why that would be more efficient.